### PR TITLE
Accessibility: Portfolio (New PR)

### PR DESCRIPTION
**Pull request by sameedmohsin28**
My coding partner has merged the previous pull request by mistake. So I have opened this new pull request for code review. For commit history of pair programming please [click here](https://github.com/sameedmohsin28/sameedmohsin28.github.io/pull/2/commits).

### Changes for better accessibility 
---

**Report listing all points that have been checked:**

- Page title: No issues found

- Image text alternatives: Added alt text in the following lines
1.  https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L20
2. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/7b6496c54594a6282f8ea6b8e28d30ce4fe38961/index.html#L43
3. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/7b6496c54594a6282f8ea6b8e28d30ce4fe38961/index.html#L48
4. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/7b6496c54594a6282f8ea6b8e28d30ce4fe38961/index.html#L53
5. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/a1cc51d14957ca9201e97f8650d7ed6bfcfa6cd3/index.html#L58
6. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/7b6496c54594a6282f8ea6b8e28d30ce4fe38961/index.html#L63
7. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L73
8. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L111
9. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L156
10.  https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L201
11. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L264
12. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L269
13. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L274
14. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L279
15. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L284

- Text headings: Set appropriate headings in the following lines
1. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L31-L33
2. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L337

- Color contrast: No issues found (It was not changed to stick to the Figma template).

- Resize: No issues found

- Interaction: Addes a label to the form fields in the following lines and set them to `visibility: hidden;` to match it to the Figma template.
1. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L343
2. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L347
3. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/index.html#L351
4. https://github.com/sameedmohsin28/sameedmohsin28.github.io/blob/237c202456315b3f81df0b48e3eb1f596c39aa61/style.css#L480-L482

- Moving content: No issues found

- Multimedia: No issues found

- Basic structure of the page: No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -326,29 +326,29 @@ <h3 class="skills-heading">
...
         <section class="contact-form-page">
+            <a name="contact-page"></a>
             <div class="contact-form-background"></div>
             <img src="./Images/1ellipse.png" alt="" class="ellipse-image">
             <img src="./Images/2upRectangle.png" alt="" class="up-rectangle">
             <img src="./Images/3bluecircle.png" alt="" class="blue-circle">
             <img src="./Images/4downRectangle.png" alt="" class="down-rectangle">
             <img src="./Images/5whiteEllipse.png" alt="" class="white-circle">
-            <h2 class="contact-form-heading">Contact me</h2>
+            <h1 class="contact-form-heading">Contact me</h1>
```
Comment (anagudelogu) [inline index.html]: - [ ] Great job applying those accessibility best practices to your project! 💯 However, I suggest you revert this change. Ideally, you would want to have only one `<h1>` element for each page, this tag is reserved for the title that describes better the content of the page (The heading one which you already changed). It serves as an index for search engines to find your page, so please consider reverting this change! 🤓 🔧

File: index.html [hunk 2]
```diff
@@ -326,29 +326,29 @@ <h3 class="skills-heading">
         </section>
 
         <!-- ======================== Contact Page ======================== -->
-        <a name="contact-page"></a>
         <section class="contact-form-page">
+            <a name="contact-page"></a>
```
Comment (anagudelogu) [inline index.html]: - [ ] I also noticed that you have this empty `<a>` tag inside your Works, About, and Contact sections. I don't know the purpose of it, but assistive technologies such as screen readers can get misleading information from this tag, which is suggesting that there is a link without a reference. Please avoid using empty elements! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @sameedmohsin28, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

- [ ] 🚧 Please consider adding a LICENSE to your project, Microverse suggests that you use the MIT one, but feel free to choose one! You can follow [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to achieve this, also, don't forget to update your README, at this moment the link to your LICENSE is broken. 🤓 🔧 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (sameedmohsin28) [reply index.html]: Hi @anagudelogu,

Thanks for your detailed review. I have done this to access the Works, About, and Contact sections from the navigation bar. If there is a better approach for doing so, kindly enlighten me with it.

https://user-images.githubusercontent.com/122236545/215921508-5664e43d-b10b-4bfa-a1d3-198a1d4e83d8.mp4

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @sameedmohsin28  👋

Great work nailing the project. I would like to express my deep appreciation for the time and effort you dedicated to this project 💪

**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks professional, well done!
-  ✅ Commits are descriptive, good work.
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (anagudelogu) [reply index.html]: Hi @sameedmohsin28, sorry for the late response. When you use a hash value like `#random_text` inside the `href` attribute of an anchor `<a>` tag, you're pointing that link to the element with this id, in this case: `random_text` so you don't need empty `<a>` tags, instead, just define the `id` of the container element as the one you have in the `href` and the link will work fine! 🤓 🔧

**Pull request merged**